DESCRIPTION

```diff
@@ -6,11 +6,11 @@ Description: A set of tools to support writing various documents
     The most common use will be for writing and compiling theses or thesis chapters,
     as drafts or for examination with all the correct preamble content.
     However, the package also offers the creation of html presentation slides via 'xaringan',
-    and, for course instructors, also the ability to create a syllabus as a PDF.
+    and, for course instructors, the ability to create syllabi.
```

Comment:
Mention problem sets?

Reply:
Done, also mentioned the bibliographical analysis toolset